Prototope/Layer+Layout.swift

```diff
@@ -66,6 +66,30 @@ extension Layer {
 	}
 	
 	
+	/** Moves the receiver so that its left side is aligned with the left side of its parent layer. */
+	public func moveToLeftSideOfParentLayer(margin: Double = 0.0) {
+		if let parent = self.parent {
+			self.frameMinX = -margin
```

Comment:
Right, yeah, I understood the logic for siblings but it only just now clicked for me that it makes sense for the margin in `parent` methods to mean “margin towards the centre of the layer”.

Made the change.